Harvest.Web/ClientApp/src/App.tsx

```diff
@@ -22,6 +22,7 @@ import { InvoiceListContainer } from "./Invoices/InvoiceListContainer";
 import { TicketCreate } from "./Tickets/TicketCreate";
 import { TicketsContainer } from "./Tickets/TicketsContainer";
 import { TicketDetailContainer } from "./Tickets/TicketDetailContainer";
+import 'react-bootstrap-typeahead/css/Typeahead.css';
```

Comment:
why do we need this now where we didn't before?  It should already be in the index.tsx page and thus doesn't need to be declared twice.

Reply:
Sorry didn't notice that. I will get rid of it.